cpp/src/cider/exec/nextgen/context/CodegenContext.h

```diff
@@ -72,7 +72,7 @@ struct CodegenOptions {
   bool check_bit_vector_clear_opt = false;
   bool set_null_bit_vector_opt = false;
   bool branchless_logic = true;
-  bool enable_vectorize = false;
+  bool enable_vectorize = true;
```

Comment:
I think some of the code path in codegen will be disabled if we enable vectorize by default. Should we cover both of the options?

Reply:
any example? I hope existing uts can approve the correctness.

Reply:
seems some test will failed, 
```bash
The following tests FAILED:
         27 - CiderNextgenQueryRunnerTest (Failed)
         36 - CaseWhenTest (SEGFAULT)
         37 - CastTest (SEGFAULT)
         38 - CiderMathOpNextgenTest (Failed)
         39 - DateTest (Failed)
         40 - FilterTest (SEGFAULT)
         42 - PrimitiveTypeQueryNextgenTest (Failed)
         43 - StringTest (SEGFAULT)
```

Reply:
open `enable_vectorize`, all uts can pass. but some failed after removing `memset`.

Reply:
> open `enable_vectorize`, all uts can pass. but some failed after removing `memset`.

some test with variable column failed caused by offset[0] not init after removing `memset`, fixed.